assets/scripts/lobby.js

```diff
@@ -985,7 +985,7 @@ function strOfAvatar(playerData, alliance) {
   ) {
     picLink = playerData.avatarImgSpy;
   } else {
-    picLink = 'avatars/base-spy.png';
```

Comment:
Not the right line but the svgs don't have a resolution or a pixel size. Let's rename.